### ISSUE TITLE
Cant install SODA CLI on Linux

### ISSUE THREAD

**Issue by GorillaBus** [gobuffalo/docs#612]
Following the instructions on the website, when trying to install:

go get github.com/gobuffalo/pop/...

I get this error message on different computers with Linux system (Fedora, Redhat):

github.com/gobuffalo/pop imports
        github.com/cockroachdb/cockroach-go/crdb: cannot find module providing package github.com/cockroachdb/cockroach-go/crdb

The installation of SODA CLI is not possible.

**Comment by sio4** [comment]
Indeed, the installation instruction is outdated and needs to be updated. I made a PR #613.

However, I am not sure of the actual error message you faced. Could you please follow the instruction below and let me know how it works? Also, please include your environment information printed by running `buffalo info` command.

Installation:

```console
$ go install github.com/gobuffalo/pop/v6/soda@latest
```
